Change to use kernel-slim image (#123)
OpenLiberty/draft-guide-operators-openshift: README.adoc

  * guide
```

> If your cluster does not install the https://cert-manager.io[cert-manager^] yet, install it by running the following command that refers to the official https://cert-manager.io/docs/installation[cert-manager installation] documentation:

Reviewer: This doesn't make sense recommend:

```suggestion
If your cluster has not installed the https://cert-manager.io[cert-manager^] yet, install it by running the following command that refers to the official https://cert-manager.io/docs/installation[cert-manager installation] documentation:
```

> If your cluster has not installed the https://cert-manager.io[cert-manager^] yet, install it by running the following command that refers to the official https://cert-manager.io/docs/installation[cert-manager installation] documentation:

Reviewer 2: ```suggestion
If the https://cert-manager.io[cert-manager^]  is not installed on your cluster yet, install it by running the following command:
```

[role='command']
```
oc apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml
```

> 

Reviewer 2: ```suggestion
For more information, see the https://cert-manager.io/docs/installation[cert-manager installation] documentation.
```
